tagging-status/index.md

```diff
@@ -36,13 +36,13 @@ The values in the *Status* column have the following meaning:
 - `no-support` This package or class or class is incompatible with the tagging code and we do *not* believe that it will ever be supported. (**{{t-s | where: "status", "no-support" | size }}** entries across all tables)
 - `unknown` The status of this package or class is not known, because there aren't reliable tests yet. Help with testing to determine the real status is very much appreciated. (**{{t-s | where: "status", "unknown" | size }}** entries across all tables)
 
-To use packages or classes together with the tagging code it is (nearly) always necessary to load at least `phase-III`of the tagging code, i.e., `testphase=phase-III` in `\DocumentMetadata`. To save space in the tables, this is not explicitly mentioned below. However, if a package or class requires other settings there is an explicit remark in the comments column, e.g., `Tagging support: phase-III, table` which means you have to specify `testphase={phase-III,table}`. If **package** is mentioned at this point it means that the package itself provides the necessary tagging support and not one of the modules in `latex-lab`. If known, **package** will link to the repository used by the package maintainer for tagging support.
+To use packages or classes together with the tagging code it is (nearly) always necessary to enable tagging with  `\DocumentMetadata{tagging=on}`. However, if a package or class requires other settings there is an explicit remark in the comments column. If **package** is mentioned at this point it means that the package itself provides the necessary tagging support and not one of the modules in `latex-lab`. If known, **package** will link to the repository used by the package maintainer for tagging support.
```

Comment:
we might have to mention (in a note or so) that right now data might still show individual testphase modules